frontend/hub/remote-registries/RemoteRegistries.tsx

```diff
@@ -18,7 +18,7 @@ export function RemoteRegistries() {
     toolbarFilters,
     tableColumns,
   });
-  const toolbarActions = useRemoteRegistriesActions();
+  const toolbarActions = useRemoteRegistriesToolbarActions();
```

Comment:
why "toolbar actions"?

This seems misleading because the actions are not only available in toolbars, there's the list entries kebab too.

Didn't we have a convention to use `useFoosActions` for the list screen and `useFooActions` for the detail screens?